app/views/sites/_form.html.erb

```diff
@@ -1,30 +1,39 @@
-<%= form_for(@site, url: site_path) do |f| %>
-  <% if @site.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(@site.errors.count, "error") %> prohibited this site from being saved:</h2>
+<div class="tab-content">
```

Comment:
I wonder if the tab panel belongs up in `sites/edit` where we're setting up the tabs? I think it'd be less confusing, and let us reuse this partial in non-tab contexts if we wanted.